src/main/java/opendota/Parse.java

```diff
@@ -104,6 +104,16 @@
 		public Integer towers_killed;
 		public Integer roshans_killed;
 		public Integer observers_placed;
+        public Integer PicksAndBans;
```

Comment:
can you use snake_case for these for consistency?

Reply:
Ok. As you can probably tell Im pretty new to programming I had to google snake_case and camel_case.